client/src/App.js

```diff
@@ -1,23 +1,38 @@
 import React from 'react';
 import './App.css';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+
 import Header from './components/Header/Header';
+import Monument from './components/Monument/Monument';
 
 function App() {
-  return (
-    <div className="App">
-      <Header/>
-      <div className="gmaps">
-        <div className="mapouter">
-          <div className="gmap_canvas">
-            <iframe id="gmap_canvas"
-                    title="gmaps-iframe"
-                    src="https://maps.google.com/maps?q=lincoln%20memorial&t=&z=13&ie=UTF8&iwloc=&output=embed"
-                    frameBorder="0" scrolling="no" marginHeight="0" marginWidth="0"></iframe>
-            Google Maps Generator by <a href="https://www.embedgooglemap.net">embedgooglemap.net</a></div>
+    return (
+        <div className="App">
+            <Router>
+                <Header/>
+                <div style={{padding: '0 4rem'}}>
+                    <Route path="/map">
+                        <div className="gmaps">
+                            <div className="mapouter">
+                                <div className="gmap_canvas">
+                                    <iframe id="gmap_canvas"
+                                            title="gmaps-iframe"
+                                            src="https://maps.google.com/maps?q=lincoln%20memorial&t=&z=13&ie=UTF8&iwloc=&output=embed"
+                                            frameBorder="0" scrolling="no" marginHeight="0" marginWidth="0"/>
+                                    Google Maps Generator by <a href="https://www.embedgooglemap.net">embedgooglemap.net</a></div>
+                            </div>
+                        </div>
+                    </Route>
+                    <Route exact path="/">
+                        <div style={{display: 'flex', justifyContent: 'center'}}>
+                            <h1>Welcome!</h1>
+                        </div>
+                    </Route>
+                    <Route path="/monuments/:monumentId" component={Monument}/>
```

Comment:
This is where we route to the monument page

Reply:
Maybe I don't understand React enough....why isn't that in a separate file?

Reply:
What do you mean? It is in a separate file, it's rendering the `Monument` component which is `components/Monument/Monument.js`

Reply:
So our App.js will be updated with new Routes each time we add one?

Reply:
We could keep it in App.js or make a specific component for it, but I think it makes sense to do it right in App.js (in the future, the routes will probably all render a specific component, instead of some html direct in App.js like I currently have for the interactive map, so it'll be more compact and clearer)